### ISSUE TITLE
followers [view] not adjust height when hidden views

### ISSUE THREAD

**Issue by buiminhhuy** [andreamazz/AMScrollingNavbar#246]
navigationController.followScrollView(tableView, delay: 50.0, followers: [toolbar])
This method have problem about calculate height.

**Comment by andreamazz** [comment]
@buiminhhuy 
can you please be more specific?

**Comment by buiminhhuy** [comment]
![image](https://cloud.githubusercontent.com/assets/568664/25479712/06c0b14e-2b70-11e7-99cc-f1f1d323a7b4.png)
![image](https://cloud.githubusercontent.com/assets/568664/25479767/38c7b188-2b70-11e7-8228-db5a61ae00c7.png)

**Comment by andreamazz** [comment]
Yeah, that's hardly helpful. What are those views? A table? With headers?  
How am I supposed to triage an issue if you don't provide the means to reproduce it?

**Comment by buiminhhuy** [comment]
I have 3 views: firstView, secondCollectionView, thirdCollectionView. I set followers[firstView, secondCollectionView]
I set navigationController.followScrollView(thirdCollectionView, delay: 50.0, followers: [firstView, secondCollectionView]
I think this problem, maybe I set wrong context.
